src/catalogue/items/catalogueItemsDialog.component.test.tsx

```diff
@@ -846,17 +846,10 @@ describe('Catalogue Items Dialog', () => {
       expect(costHelperText).toBeInTheDocument();
       expect(daysToReplaceHelperText).toBeInTheDocument();
 
-      expect(
-        screen.getByText(
-          'Please choose a manufacturer, or add a new manufacturer'
-        )
-      ).toBeInTheDocument();
```

Comment:
Why has this been removed?

Reply:
Same reason as above - it is no longer clearable